### ISSUE TITLE
My TOHKBD stopped working

### ISSUE THREAD

**Issue by xoquox** [dirkvl/TOHKBD#3]
Hello,
my TOHKBD stopped working.
I tested all the contacts and it looks like everything is fine with the TOH. The EPROM on the TOH works, and it reinstalls the software.
So i looked up the harbour-tohkbd2 service. All starts fine, but when i put on the Keyboard, it gives me this message: 
Jul 10 22:25:01 Jolla harbour-tohkbd2[12142]: Presence detected, turning power on
Jul 10 22:25:01 Jolla harbour-tohkbd2[12142]: VDD control request - turn on
Jul 10 22:25:02 Jolla harbour-tohkbd2[12142]: Error: reset() failed

So, could it be a problem with the TCA8424 or could it be a software problem?
I tested the TOHKBD on two Jolla's , the keyboard used to work on both befor.

I hope to hear from you soon,
greedings 
Christian

**Comment by dirkvl** [comment]
What do you get from probing the i2c bus in I2Ctool?

2015-07-10 22:36 GMT+02:00 Christian notifications@github.com:

**Comment by xoquox** [comment]
Probing with I2Ctool gets me [Black] on adresss 04-07 and [Blue] on adresses 50-57
So no ACK on any EEPROM :(

**Comment by xoquox** [comment]
I also tested the voltage regulator on the TOH, and it put out 1,8V.  So maybe the EEPROM on TOH stopped working?

**Comment by dirkvl** [comment]
ehhh i dont know. send it back and ill fix it. my address is on
funkyotherhalf.com, include a note with the issue and a return address.

dirk

2015-07-11 11:09 GMT+02:00 Christian notifications@github.com:

**Comment by Bulkin** [comment]
Any news on this issue? I have the same problem after a phone reboot.

**Comment by dirkvl** [comment]
Huh? You sent it back right? You got no 'it is repaired' mail? :s

2015-08-15 0:42 GMT+02:00 Vasili Bulkin notifications@github.com:

**Comment by Bulkin** [comment]
No, I didn't send it back, I only received it a week ago. Anyway, the EPROM isn't really that important, right? It's just for plug-and-play support.

**Comment by dirkvl** [comment]
the eeprom on the casing side is crucial, the eeprom on the keyboard side
not so much

2015-08-19 1:24 GMT+02:00 Vasili Bulkin notifications@github.com:
